ods_ci/utils/scripts/fetch_tests.py

```diff
@@ -48,12 +49,11 @@ def get_repository(test_repo):
         if "error" in ret.lower():
             # actual error gets printed during "execute_command"
             raise Exception("Failed to clone the given repository")
+    elif not os.path.exists(test_repo):
```

Comment:
anyways, what do you think about this `else: if:` warning? I was unsure when I was enabling it, whether it is actually helpful or whether it is just dictating a bit unnatural coding style

I guess in this case I like the changed code a little better, so maybe the warning is good

Reply:
In this particular case, I would have preferred having `else: if:` and avoided the next `else`: which would have falled under the same `else:` because both things were about `local path`.

Apart from this minimal thing, I think it should be ok most of the cases. We can judge over time, we don't write much pure python code

Reply:
> we don't write much pure python code

that may very well change in the mid-term future ;p